src/components/navbar/useNavbar.tsx

```diff
@@ -175,13 +183,29 @@ const useNavbar = (
         } catch (error) {
             // eslint-disable-next-line no-console
             console.error(error);
+            setHasDownloadError(true);
+            const toastNotification = {
+                id: 'document-export',
+                message: `Export failed`,
+                type: ToastVariant.NEGATIVE,
+            };
+
+            addNotification(toastNotification);
         } finally {
             updateDownloadState({ [extension]: false });
         }
         hideDownloadPanel();
     };
 
-    return { navbarItems, showDownloadPanel, hideDownloadPanel, isDownloadPanelVisible, handleDownload };
+    return {
+        navbarItems,
+        showDownloadPanel,
+        hideDownloadPanel,
+        isDownloadPanelVisible,
+        handleDownload,
+        setHasDownloadError,
+        hasDownloadError,
```

Comment:
do we still need this field?

Reply:
removed 62d831f69246c12f4554fe0e094958d157539ac2